### DISCUSSION TITLE
Disable fruit spawn on the same position as the snake body

### DISCUSSION THREAD

**Discussion by rajatnai49** [rajatnai49/sneakySnake#1] [Q&A] [answered]
You have to change fruit spawn like that by which fruit never spawn on the snake body.

**Comment by kunjgit** [comment, marked as answer]
What I have done is I have checked the apple spawn co ordinates and taken it to validity check for the snake body and once we are confirm that the point is not on the snake body then only we are moving with the further logic

Checkout my pull request that I have done to remove the conflict and if it's great merge the same 
#2 

**Thank you🙏**
